scripts/zfs.sh

```diff
@@ -57,20 +64,158 @@ while getopts 'hvu' OPTION; do
 	esac
 done
 
+kill_zed() {
+	if [ -f "$ZED_PIDFILE" ]; then
+		PID=$(cat "$ZED_PIDFILE")
+		kill "$PID"
+	fi
+}
+
+check_modules() {
+	local LOADED_MODULES=""
+	local MISSING_MODULES=""
+
+	for KMOD in $KMOD_SPL $KMOD_SPLAT $KMOD_ZAVL $KMOD_ZNVPAIR \
+	    $KMOD_ZUNICODE $KMOD_ZCOMMON $KMOD_ICP $KMOD_ZFS; do
+		local NAME=$(basename "$KMOD" .ko)
```

Comment:
`shellcheck` is going to have you break this out into the following:
```
local NAME
NAME=$(basename "$KMOD" .ko)
```
You can opt to use the `# shellcheck disable` directive to get around it as well.